MCprep_addon/MCprep_resources/mcprep_data_update.json

```diff
@@ -869,7 +869,7 @@
 			"absorbing_hardcore_half": "gui/sprites/hud/heart/absorbing_hardcore_half",
 			"absorbing_hardcore_half_blinking": "gui/sprites/hud/heart/absorbing_hardcore_half_blinking",
 			"absorption": "mob_effect/absorption",
-			"acacia": "entity/chest_boat/acacia",
+			"acacia": "entity/signs/acacia",
```

Comment:
Seems strange that `chest_boat` was changed to `signs`. I think we should investigate this further.

Reply:
I also agree... but these are completely auto generated. So likely more to do with alphabetical orderings changing or folder names within the jar files of Minecraft itself. If anything, we should expand the rulesets of things to just outright ignore or more intentionally sub-category.